index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500;700&display=swap" rel="stylesheet">
+    <link
+      href="//db.onlinewebfonts.com/c/23c0fcab84d99da0de762de7e220a6e1?family=Europa"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="http://fonts.cdnfonts.com/css/eurostile-extended"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  </head>
+  <body>
+    <nav>
+      <img src="./images/mf-logo.svg" alt="" class="brand-img" />
+
+      <div class="nav-right">
```

Comment:
div not needed

Reply:
don't I need this in order to group the button and the mentorship so that they stay near each other?

if there is another way, please tell me

Reply:
you can apply the changes directly on the "nav-text" and from that point change whats needed for the "nav-buttton" to stay near each other.

Reply:
or you can use flex
https://css-tricks.com/snippets/css/a-guide-to-flexbox/